benchmarks/benches/complicated_maud.rs

```diff
@@ -0,0 +1,102 @@
+#![feature(plugin, test)]
+#![plugin(maud_macros)]
+
+extern crate maud;
+extern crate test;
+
+use maud::{Markup, PreEscaped};
+
+#[derive(Debug)]
+struct Entry {
+    name: &'static str,
+    score: u16,
```

Comment:
I was going off rust-lang/rust#22240 which recommends `u32` here.

It's not a performance/alignment issue, but a style one. From what I've seen, 32 bits is the "default" size, and what you should go for first.